### DISCUSSION TITLE
Auto organize the open windows

### DISCUSSION THREAD

**Discussion by jfbaro** [nextapps-de/winbox#158] [Ideas]
Hello!!

Congrats on the great piece of software!

Would it be possible to offer a (right click?) way of asking the HOST application to organize all open windows? Something like:

"Show Windows Side by Side"
"Show Windows Stacked"

?

Thanks !

**Comment by ts-thomas** [comment]
Yes of course. Everything we can imagine is also possible, the only limitation is our imagination :)
This could be done by some custom code. When creating a Winbox instance it needs to be stored in an Array (removing from this Array onclose). Then you just need a button and apply a listener which runs through this array and move each window to the desired position (width/height/x/y needs to be calculated dependent on window count and available screen size).
